### PR TITLE
fix: handled error from get_username gracefully instead of panicking

### PR DESCRIPTION
**Pull request by ashupednekar**
- instead of unwrapping on get_username result, propagating the error properly to main so that it gracefully fails instead of panicking

### CODE REVIEW

File: src/config/mod.rs [hunk 1]
```diff
@@ -35,12 +36,12 @@ fn get_config_file_name() -> String {
 /// ```
 /// assert!(check_config_exists());
 /// ```
-fn check_config_exists() -> bool {
-    fs::metadata(get_config_file_name()).is_ok()
+fn check_config_exists() -> Result<bool> {
+    Ok(fs::metadata(get_config_file_name()).is_ok())
```
Comment (amritghimire) [inline src/config/mod.rs:40]: Why is this necessary if Ok is returned always?

**Review by amritghimire — CHANGES_REQUESTED** [review]

File: src/config/mod.rs (hunk 1 above)
Comment (ashupednekar) [reply src/config/mod.rs:40]: It's just to propagate the error from create config all the way to main, so it doesn't panic 

Also, in the future there's value in adding a proper cli library like clap

**Comment by amritghimire** [comment]
Can you fix the rust fmt?

**Comment by ashupednekar** [comment]
> Can you fix the rust fmt?
done :+1:

**Review by amritghimire — APPROVED** [review]
